binders/kafka-binder/spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaConsumerProperties.java

```diff
@@ -210,6 +210,16 @@ public enum StandardHeaders {
 	 */
 	private boolean reactiveAtMostOnce;
 
+	/**
+	 * Consumer priority level. NOTE: Kafka does not natively support consumer priority.
+	 * This property is provided for consistency across binders but has no effect in Kafka.
+	 * For even message distribution across servers, use partition assignment strategies
+	 * or create separate bindings with concurrency=1.
+	 * Default: -1 (not supported)
+	 * @since 4.2
+	 */
+	private int consumerPriority = -1;
```

Comment:
We don't need this for consistency. It is actually more confusing given the comment. I will remove it